### PR TITLE
DATA_STORE_DIRECTORY

### PR DESCRIPTION
**Pull request by SViksha**
Added "DATA_STORE_DIRECTORY=/opt/mithril/mithril-signer/store" in environment file

### CODE REVIEW

File: docs/root/manual/getting-started/run-signer-node.md [hunk 1]
```diff
@@ -132,6 +132,7 @@ Replace this value with the correct user. We assume that the user used to run th
   * `CARDANO_NODE_SOCKET_PATH=/cardano/ipc/node.socket`: replace with the path to the IPC file (`CARDANO_NODE_SOCKET_PATH` env var)
   * `CARDANO_CLI_PATH=/app/bin/cardano-cli`: replace with the path to the `cardano-cli` executable
   * `STAKE_STORE_DIRECTORY=/opt/mithril/mithril-signer/store/stake_db`: replace with the path to a folder where the **Mithril Signer** will store its data (`/opt/mithril/mithril-signer/store` e.g.)
```
Comment (jpraynaud) [inline docs/root/manual/getting-started/run-signer-node.md:134]:
```suggestion
```
This is not needed any more

File: docs/root/manual/getting-started/run-signer-node.md [hunk 2]
```diff
@@ -147,6 +148,7 @@ RUN_INTERVAL=60000 DB_DIRECTORY=/cardano/db
 CARDANO_NODE_SOCKET_PATH=/cardano/ipc/node.socket
 CARDANO_CLI_PATH=/app/bin/cardano-cli
 STAKE_STORE_DIRECTORY=/opt/mithril/mithril-signer/store/stake_db
```
Comment (jpraynaud) [inline docs/root/manual/getting-started/run-signer-node.md:150]:
```suggestion
```
Same here

File: docs/root/manual/getting-started/run-signer-node.md [hunk 3]
```diff
@@ -132,6 +132,7 @@ Replace this value with the correct user. We assume that the user used to run th
   * `CARDANO_NODE_SOCKET_PATH=/cardano/ipc/node.socket`: replace with the path to the IPC file (`CARDANO_NODE_SOCKET_PATH` env var)
   * `CARDANO_CLI_PATH=/app/bin/cardano-cli`: replace with the path to the `cardano-cli` executable
   * `STAKE_STORE_DIRECTORY=/opt/mithril/mithril-signer/store/stake_db`: replace with the path to a folder where the **Mithril Signer** will store its data (`/opt/mithril/mithril-signer/store` e.g.)
+  * `DATA_STORE_DIRECTORY=/opt/mithril/mithril-signer/store`: replace with the path to a folder where the **Mithril Signer** will store its data (`/opt/mithril/mithril-signer/store` e.g.)
```
Comment (jpraynaud) [inline docs/root/manual/getting-started/run-signer-node.md:135]:
```suggestion
  * `DATA_STORES_DIRECTORY=/opt/mithril/mithril-signer/store`: replace with the path to a folder where the **Mithril Signer** will store its data (`/opt/mithril/mithril-signer/store` e.g.)
```

File: docs/root/manual/getting-started/run-signer-node.md [hunk 4]
```diff
@@ -147,6 +148,7 @@ RUN_INTERVAL=60000 DB_DIRECTORY=/cardano/db
 CARDANO_NODE_SOCKET_PATH=/cardano/ipc/node.socket
 CARDANO_CLI_PATH=/app/bin/cardano-cli
 STAKE_STORE_DIRECTORY=/opt/mithril/mithril-signer/store/stake_db
+DATA_STORE_DIRECTORY=/opt/mithril/mithril-signer/store
```
Comment (jpraynaud) [inline docs/root/manual/getting-started/run-signer-node.md:151]:
```suggestion
DATA_STORES_DIRECTORY=/opt/mithril/mithril-signer/store
```

**Review by jpraynaud — CHANGES_REQUESTED** [review]
Thanks @SViksha for finding out this issue :+1:

**Comment by jpraynaud** [comment]
Hi @SViksha, are you ok if I apply the proposed modifications above and then merge the PR?

**Comment by SViksha** [comment]
> Hi @SViksha, are you ok if I apply the proposed modifications above and then merge the PR?

sure, looks good to me

**Review by jpraynaud — APPROVED** [review]
LGTM :+1:

**Pull request merged**